provider/cmd/pulumi-resource-command/schema.json

```diff
@@ -471,6 +471,11 @@
                     "description": "SSH Agent socket path. Default to environment variable SSH_AUTH_SOCK if present.",
                     "type": "string"
                 },
+                "dialErrorLimit": {
+                    "default": 10,
+                    "description": "Max allowed errors on trynig to dial the remote host. 0 set count to unlimited. Default value is 10",
```

Comment:
Should `-1` be the "infinite" option, so that `0` can be the "don't retry" option?

Reply:
yeah -1 was my first choice actually and then moved to 0 so totally fine. Already fixed